src/consumerIncentives/saga.ts

```diff
@@ -77,7 +77,7 @@ export function* claimRewardsSaga() {
       const claimRewardFn = isSuperchargePendingRewardsV2(rewards) ? claimRewardV2 : claimReward
       receivedRewards = yield all(
         // @ts-ignore remove this ignore when upgrading to TS 4.2+ https://github.com/microsoft/TypeScript/issues/36390
-        rewards.map((reward, index) => call(claimRewardFn, reward, index, baseNonce))
+        rewards.slice(0, 1).map((reward, index) => call(claimRewardFn, reward, index, baseNonce))
```

Comment:
i added this so that my rewards are not all claimed at once in the existing system, since once they're claimed you can't test with them anymore! i had a few rewards lined up so this helped me, leaving it here in case it helps someone else (but feel free to remove also)